wordpress-mobile/WordPress-iOS
# Remove empty box flashes

Fixes #14080 

To test:
1. Install and launch the application.
2. Tap on "Log In".
3. Log in to the app using valid credentials.
4. Open "My site" tab.
5. Tap on "Menus".
6. Select or add new menu.
7. Tap on New item and list will open.
8. Tap on any option from the list (eg. Page,Link, Tag, Post,Product)
9. Should't be white box in light theme case and dark box in dark theme case.

PR submission checklist:
